matrix/common/etl/transformers/cell_expression.py

```diff
@@ -136,14 +139,23 @@ def _parse_10x_bundle(self, bundle_dir):
             h.update(str(lane_index).encode())
             cell_key = h.hexdigest()
 
-            # decide if this is a lucky cell
-            is_lucky = str(ord(cell_key[-1]) % 5 == 0)
+            cell_to_barcode[cell_key] = barcode
 
-            expression_line = '|'.join(
-                [cell_key,
-                 gene,
-                 "Count",
-                 str(v)]) + '\n'
+            if cell_key not in cell_gene_counts:
```

Comment:
This code is a little difficult to understand but fine for revisiting it at a later time

Reply:
Agreed, the actual metadata transformations could use a little clean up, especially as they're only going to get more complex with updates to metadata schema.